onnxruntime/test/providers/cpu/tensor/scatter_op_test.cc

```diff
@@ -32,6 +33,29 @@ TEST(ScatterOpTest, WithoutAxis) {
   test.Run();
 }
 
+TEST(ScatterElementsOpTest, WithoutAxis) {
+  OpTester test("ScatterElements", ScatterElements_ver);
+
```

Comment:
Could the two versions of each test be implemented in one common function with the op name and version passed in?

Reply:
Done.